docs/init-config.sample.yaml

```diff
@@ -16,23 +16,23 @@ Mempool:
     Capacity: 65535
   GUIDER:  # TX modified Interest guiders (forwarder only)
     Capacity: 65535
-  INT:  # TX Interests (trafficgen only)
+  INT:  # TX Interests (traffic generator only)
     Capacity: 65535
-  DATA:  # TX Data hader (trafficgen only)
+  DATA:  # TX Data header (traffic generator only)
     Capacity: 65535
-  PAYLOAD:  # TX Data payload (trafficgen only)
+  PAYLOAD:  # TX Data payload (traffic generator only)
     Capacity: 255
 
 # LCORE ALLOCATION
 # This is a mapping from LCore role to its reservations.
 # Roles for ndnfw-dpdk: RX, TX, CRYPTO, FWD.
 # Roles for ndnping-dpdk: RX, TX, SVR, CLIR, CLIT.
 # Within each role:
-# 'lcores' is a list of lcores reserved for a role.
-# 'pernuma' is a map of { NumaSocket => max number of lcores } for a role.
-# Leaving this section blank results in default allocation.
+#  - 'lcores' is a list of lcores reserved for that role.
+#  - 'pernuma' is a map of { NumaSocket => max number of lcores } for that role.
+# Leaving this section blank results in automatic allocation.
```

Comment:
is "automatic" accurate here?

Reply:
Yes, the allocation is automatic, but only if the section is entirely blank.
If any role has a manual allocation, all roles must have manual allocation.